locust/load-test/example.yaml

```diff
@@ -10,7 +11,9 @@
   offset: 1
   method: GET
   headers: ''
+  alias: example 1
```

Comment:
I don't think this should be mandatory, maybe leave it optional, and default to query if it's not set.

Reply:
Fixed.